dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -153,7 +153,9 @@ export default defineComponent({
         ][]) {
           if (subCategoryObject == null) continue;
           for (const [kpiKey, kpiValue] of Object.entries(subCategoryObject)) {
-            if (kpiValue == null) continue;
+            if (kpiValue == null) {
+              subCategoryObject[kpiKey] = "";
```

Comment:
Why not change the kpi value

Reply:
we change the kpiValue value in the object under the key subCategoryObject[kpiKey]